### DISCUSSION TITLE
used vs2019 for mac Edit OC

### DISCUSSION THREAD

**Discussion by OlziiNaran** [OrchardCMS/OrchardCore#9578] [Q&A]
I used VS2019 for mac to compile OC, and the "Go to Declaration"/"Go to Implementation"/"Go to base Member" of the class is invalid. Is it a VS environment problem? Or is it an OC code issue? 

Views:
<img width="570" alt="Screen Shot 2021-05-29 at 02 12 34" src="https://user-images.githubusercontent.com/39524174/120025570-74267d80-c023-11eb-942d-e6fec3e3882c.png">
<img width="516" alt="Screen Shot 2021-05-29 at 02 12 18" src="https://user-images.githubusercontent.com/39524174/120025609-87394d80-c023-11eb-9ceb-6aa9805fcfa6.png">
<img width="507" alt="Screen Shot 2021-05-29 at 02 11 57" src="https://user-images.githubusercontent.com/39524174/120025646-97e9c380-c023-11eb-8e20-97a310b58433.png">

**Comment by Skrypt** [comment]
Intellisense most likely.

**Reply by OlziiNaran** [reply]
???

**Reply by Skrypt** [reply]
There is no issue with this on Visual Studio under Windows or Visual Studio Code (omnisharp). You could try using Visual Studio Code under Mac OS.

Also worth trying : https://stackoverflow.com/questions/57474785/visual-studio-2019-go-to-definition-and-intellisense-not-working

**Reply by OlziiNaran** [reply]
Tryed all, but invalid
